c7n/resources/cloudtrail.py

```diff
@@ -206,3 +206,35 @@ def process(self, resources):
                 client.start_logging(Name=r['Name'])
             else:
                 client.stop_logging(Name=r['Name'])
+
+
+@CloudTrail.action_registry.register('delete')
+class DeleteTrail(BaseAction):
+    """ Delete a cloud trail
+
+    :Example:
```

Comment:
example is lower case I believe for restructured text and unless there's some text following it can omitted.

Reply:
Thanks! Fixed it to lower case